contents/handbook/people/team-structure/core-experience.md

```diff
@@ -15,24 +15,34 @@ hideAnchor: true
 
 ## Mission
 
-To create the easiest way to discover insights about products and its users
+To create the easiest experience to discover insights about products and its end users.
 
 ## Responsibilities
+List below is prioritized.
 
-- Extending feature set as suggested by requests, by our own ideas, and by need for parity with other platforms
-- Maintaining data quality and clarity
-- Ensuring performant and clear user experience across all analytics functionality
+1. Reaching parity in terms of features, performance and user experience with leading product analytics solutions, including:
+   1. Maintaining data quality and consistency.
+   2. Ensuring performant and clear user experience across all analytics functionality.
+   3. Extending core feature set as informed by dogfooding, user feedback, experimentation and strategic decisions.
+   4. Extending user experience to enable product analytics for non-technical users. 
+2. Maintaining (and improving where applicable) supporting activities for the core experience (this includes: cohorts, persons, sessions *[not session recording]*, annotations, settings, authentication, account management). 
 
 
 ## Customer
 
-- Any PostHog user, whether they're an engineer or a product manager, should be able to effectively analyze their product.
+Any PostHog user, whether they're an engineer, product manager or business person, should be able to effectively analyze their product.
 
 ## Output metrics
+Our output metrics can be tracked in [Dashboard #3307](https://app.posthog.com/dashboard/3307).
 
-- Retention per feature
+#### WAU/MAU ratio
+Ratio of weekly active users versus monthly active users (rolling period) where an active user is someone who *Discovered a Learning* during the period. This signals core product usage without being affected by activation or acquisition activities.
```

Comment:
I think we should define *discovered learning* here.

Reply:
It's defined elsewhere, I don't want to repeat the definition to avoid inconsistencies if we update it. Though a link it's absolutely needed here. @kpthatsme what should we use as the source of truth for this definition?

Reply:
I don't think there's a definition today in any public channels, just our internal repos.

As far as a definition, for a short one, we can define it as a "meaningful interaction using insights on their data".

I don't think we need to go deeper on it since there are quite a few events we look at, and I don't know how much value that adds to the handbook. We can link to the action if we want to include the whole thing for those that have access.

In terms of where we should put this – do we think it belongs on a team page? I'm okay with you adding a short blurb to your page for now. I do wonder if it belongs somewhere more central ie. growth strategy https://posthog.com/handbook/growth/strategy